src/simulator/simulator.hpp

```diff
@@ -100,7 +109,19 @@ class Simulator : public ISimulator
 private:
     auto find_model(std::string name);
 
-    std::ofstream log_out{".log/" + std::to_string(core::mpi::rank()) + ".out"};
+    auto static log_file_name()
+    {
+        if (auto log = core::get_env("PHARE_LOG"); log == "RANK_FILES")
+            return ".log/" + std::to_string(core::mpi::rank()) + ".out";
```

Comment:
yes, that is something that is annoying, and I can handle it

Reply:
already done it seems
`https://github.com/PHAREHUB/PHARE/blob/master/pyphare/pyphare/simulator/simulator.py#L190`

Reply:
I think this can be let to another issue tbd later, in particular it could be moved from the simulator to the simulation object, so that phare-exe also benefits from that.